Header header
geometry_msgs/Point R_target
geometry_msgs/Point L_target